starter-code/index.js

```diff
@@ -1,10 +1,33 @@
-var SortedList = function() {};
+var SortedList = function(items, length) {
+    this.items = [];
+    this.length = 0;
+};
 
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+SortedList.prototype.add     = function(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => {
+        return a - b;
+    });
+    this.length++;
+};
+SortedList.prototype.get     = function(pos) {
```

Comment:
We have to check if we have elements in the array and also how long our array is so we know the "scope" of the array: 
```
if (pos < 1 || pos > this.length){
        throw Error("OutOfBounds");
    }
```